### DISCUSSION TITLE
Front end permissions

### DISCUSSION THREAD

**Discussion by brucebailey** [MasaCMS/MasaCMS#154] [Q&A]
Is there a way to limit the access of the front end editing to just edit certain pages. I have the permission for a group set so they can only edit certain pages, but when logged in as that user, I have access to add pages as well as get to the back end of the site. Mind you, on the back end, that user can only edit the pages they have access to, but I don't like that they can see the whole structure of the site.

Reading this in the "Site Members Permissions" suggests that they won't have access:

![image](https://user-images.githubusercontent.com/913250/211657321-a876eb21-b3bc-4fe6-817e-5cf3a11ae5d1.png)

And when logged in, they still have access here:

<img width="711" alt="image" src="https://user-images.githubusercontent.com/913250/211657371-abb96778-a292-4b42-82dc-0cee353f3b15.png">

That being said, I could very well have it set up wrong.

**Comment by brimard** [comment]
Couple of things you can check:

   - The group isn't set via the content manager permissions page to have
   access
   - Users are site members and not system users

The four or five steps needed to set perms for web editors has always
caught me up on Mura/Masa; always miss one of them. I think the key one may
be the first bullet; setting content manager permissions.

**Comment by brucebailey** [comment]
By content manager permissions, do you mean this screen?

![Screen Shot 2023-01-11 at 6 29 37 AM](https://user-images.githubusercontent.com/913250/211795524-e2042681-4964-4d0b-8412-b7f7c995710c.png)

**Comment by brimard** [comment]
I was wrong about content manager under site settings, that does need to be set.  The difference is only based on member group versus system group. Here are some older instructions, but I don't think permissioning has changed for years:

However, all System Group Users also have keys to the back-end administration area of the web site where they may be able to access the Site Manager and one or more Modules such as Components, etc. You may want to allow a specific group of users to edit content, but not necessarily access the back-end administration area of the site.

To allow a group of users the ability to make changes to content, but keep them from accessing the back-end administration area, follow these general steps:

- Create a Member Group 
- Grant Site Access/Permissions to the desired Site Member Group
- Grant Content Access/Permissions to the desired Site Member Group 

Once you've completed these steps, when a Member Group User logs in and navigate to a content item their group has been granted editing privileges to, they will see a similar toolbar as System Group Users. The main exception is they won't have access to the Site Manager.
